manifests/install.pp

```diff
@@ -4,11 +4,21 @@
 class candlepin::install {
   assert_private()
 
-  package { ['candlepin']:
-    ensure => $candlepin::version,
-  }
+  if $candlepin::container {
+    ensure_packages(['podman'])
+
+    exec { 'pull candlepin image':
+      command => "/usr/bin/podman pull ${candlepin::container_image}:${candlepin::container_version}",
+      require => Package['podman'],
+      timeout => 600,
+    }
```

Comment:
Is there any way to make this idempotent?

Reply:
Great question -- checkout the updated code with the unless that I added.